cmd/batch_ingestion.go

```diff
@@ -38,6 +38,7 @@ func RunBatchIngestion() error {
 	licenseConfig := models.LicenseConfiguration{
 		LicenseKey:             utils.GetEnv("LICENSE_KEY", ""),
 		KillIfReadLicenseError: utils.GetEnv("KILL_IF_READ_LICENSE_ERROR", false),
+		IsDevEnvironment:       utils.GetEnv("FIREBASE_AUTH_EMULATOR_HOST", "") != "",
```

Comment:
Hmm wait, this doesn't quite work.
The FIREBASE_AUTH_EMULATOR_HOST env variable is not needed for the batch ingestion jobs (or other batch jobs). As a consequence, there is no point in setting it for this job, and conversely it can be set to any non-empty value without side effects (which would bypass license checks).
This shortcut works for the server only, because setting the variable actually changes the behavior (on authentication).
Now, considering the feature entitlements currently added in NewDevLicense, this could just about work (in practice, you'd need the entitlement on the app to create the workflows, roles etc), but I'm not sure it's the best way to move forward.